src/main/scala/nl.knaw.dans.easy.deposit/EasyDepositApiApp.scala

```diff
@@ -191,22 +195,21 @@ class EasyDepositApiApp(configuration: Configuration) extends DebugEnhancedLoggi
    * @return
    */
   def writeDataMetadataToDeposit(dm: DatasetMetadata, user: String, id: UUID): Try[Unit] = for {
-    deposit <- DepositDir.get(draftsDir, user, id)
+    deposit <- getDeposit(user, id)
     _ <- deposit.writeDatasetMetadataJson(dm)
   } yield ()
 
   /**
-   * Returns the list of [[FileInfo]] objects for the dataset files in this deposit.
+   * Returns dataset files of the deposit
```

Comment:
Why did you change the description?

Reply:
Because the method didn't return anymore [FileInfo] objects, but deposit data files.